### DISCUSSION TITLE
PixiJs mask filter with background image

### DISCUSSION THREAD

**Discussion by AlekseiPinchuk** [pixijs/pixijs#8082] [Q&A]
Hello
How can I create effect like on this site (work links)

https://monopo.london/work/

I was trying implement this example, but I can't create effect on mouse enter and on mouse leave (scaling)

My trying on CodePen (https://codepen.io/alekseipinchuk/pen/NWavgvv)

Can someone help me?

.

**Comment by domis86** [comment]
Here: https://stackoverflow.com/questions/70438296/pixijs-mask-filter-with-background-image#comment124689866_70462230
https://codepen.io/domis86/pen/VwMQRGB
